src/WebJobs.Script.WebHost/Metrics/LinuxContainerMetricsPublisher.cs

```diff
@@ -251,8 +251,8 @@ public void Initialize()
         public void Start()
         {
             Initialize();
-            _processMonitorTimer = new Timer(OnProcessMonitorTimer, null, TimeSpan.Zero, _memorySnapshotInterval);
-            _metricsPublisherTimer = new Timer(OnFunctionMetricsPublishTimer, null, TimeSpan.Zero, _metricPublishInterval);
+            _processMonitorTimer = new Timer(OnProcessMonitorTimer, null, TimeSpan.FromSeconds(2), _memorySnapshotInterval);
```

Comment:
What's the impact if this is changed to 5 sec? some cold start scenarios might still take longer than 2 sec for now

Reply:
Taking this offline since some of this has dependency on other infrastructure components.

Reply:
Decided to keep it this way since making it larger doesn't seem to make much of a difference. We can revisit if this becomes an issue later.

Reply:
Yes it's ok to delay this call if needed.